spacy_loggers/wandb.py

```diff
@@ -92,6 +93,21 @@ def log_step(info: Optional[Dict[str, Any]]):
                         )
 
         def finalize() -> None:
+
+            if log_best_dir:
+                log_dir_artifact(
+                    path=log_best_dir,
+                    name="model_best",
```

Comment:
Should the name also refer to `run.id` like the models from the steps do?

Reply:
My reasoning for not doing this was that, this way, there is one `model_best` artifact per project. Each run should update that artifact, so the models of each run are still accessible via versioning.

The checkpointing uses versioning to id each checkpoint, which makes sense for that but IMO not so much for the end model of each run. So I would not refer to `run.id`, but I can see why you would want to keep things consistent, so wouldn't object if you prefer it that way.

Reply:
Ok, I can follow your reasoning :-) Let's keep it how you originally had it.